src/main/java/com/snowflake/conf/SnowflakeConf.java

```diff
@@ -36,6 +36,8 @@
       "The role to use to connect to Snowflake."),
     SNOWFLAKE_JDBC_DB("snowflake.jdbc.db", "db",
       "The database to use to connect to Snowflake."),
+    SNOWFLAKE_JDBC_WAREHOUSE("snowflake.jdbc.warehouse", "warehouse",
+      "The warehouse to use with Snowflake, if necessary."),
```

Comment:
why do we need this?

Reply:
we use information schema to get the files registered, iirc

Reply:
I'll see if I can add something like "show files" which won't use a WH, closing for now.